enable_relocalization: true
map_resolution: 0.1
map_width: 512
map_height: 512
map_origin_center: true
relocalization_map_out_filename: "/home/euclid/ROS/maps/relocalization_out.map"
occupancy_map_out_filename: "/home/euclid/ROS/maps/occupancy_out.map"
trajectory_map_out_filename: "/home/euclid/ROS/maps/trajectory_out.ppm"
relocalization_map_in_filename: "/home/euclid/ROS/maps/relocalization_in.map"
occupancy_map_in_filename: "/home/euclid/ROS/maps/occupancy_in.map"
trajectory_map_in_filename: "/home/euclid/ROS/maps/trajectory_in.ppm"
depth_of_interest_min: 0.2
depth_of_interest_max: 4.0
height_of_interest_min: -0.1
height_of_interest_max: 1.0
